usercp.php

```diff
@@ -3317,6 +3317,10 @@
 				$type = "thread";
 			}
 
+			if(empty($draft['subject']))
+			{
+				$draft['subject'] = "[".$lang->post_no_subject."]";
```

Comment:
This might be nice to be translatable, since we're trying to minimise the amount of hardcoded HTML and other text.

Reply:
I followed the way of handling empty post subject in `showthread.php`. Then I find that I should also make `[]` as translatable, you can see I haven't enclosed them with brackets in `modcp.php`.

I'll try to update them and `showthread.php` as well.

Reply:
Ok. It seems we still have a lot of hardcoded stuff scattered about in that case 😢

Reply:
Seems `no_subject` is only hardcoded in `showthread.php` before this PR. But for `RE: `... pretty a lot.

Reply:
We'll definitely do a separate PR for `RE:` then, otherwise it would make following PRs nearly impossible.